src/modules/Todos.js

```diff
@@ -0,0 +1,46 @@
+export class Todos {
+  constructor() {
+    const todoDataFromLocalStorage = JSON.parse(
+        localStorage.getItem('todoData'),
+    );
+    const todoLSExists =
+      todoDataFromLocalStorage !== undefined &&
+      todoDataFromLocalStorage !== null;
+
+    this.todoList = todoLSExists ? todoDataFromLocalStorage : [];
+  }
+
+  addTodo(todo) {
+    this.todoList.push(todo);
+    this.saveToLocalStorage();
+  }
+
+  removeTodo(idx) {
+    console.log('XX', idx);
+    this.todoList = this.todoList.filter(({index}) => index !== idx);
+    this.todoList = this.todoList.map((todo, i) => {
+      return {
+        ...todo,
+        index: i,
+      };
+    });
+    this.saveToLocalStorage();
+  }
+
+  editTodo(idx, newDesc) {
+    const todo = this.todoList.find(({index}) => index === parseInt(idx));
+    todo.description = newDesc;
+    this.todoList = this.todoList.map((todos) => {
+      if (todos.index === idx) {
+        return todo;
+      }
+      return {
+        ...todos,
+      };
+    });
+  }
```

Comment:
- [ ] The edit functionality is not working as expected, If I edit any task and refresh the page I still get the old task description it's not being updated in the local storage. Please make sure to fix this issue and users should be able to get the edited task description after the page refresh.